tests/models/test_model.py

```diff
@@ -110,6 +110,8 @@ def test_model_log():
         assert isinstance(loaded_example, pd.DataFrame)
         assert loaded_example.to_dict(orient="records")[0] == input_example
 
+        assert loaded_model.mlflow_version == mlflow.version.VERSION
```

Comment:
`from packaging.version import Version`

```suggestion
        assert Version(loaded_model.mlflow_version) == Version(mlflow.version.VERSION)
```